### DISCUSSION TITLE
No submit button?

### DISCUSSION THREAD

**Discussion by dodoburner** [codewars/codewars.com#2672] [Q&A] [answered]
I can run the tests and the attempt but there is not submit button? What could be the issue
![codewars (message redacted: screenshot suppressed)]()

**Comment by hobovsky** [comment, marked as answer]
The "Submit" button appears after you ATTEMPT and pass all the tests.
Press the ATTEMPT button. Full test suite will run, and if tests pass, the ATTEMP button will be replaced with SUBMIT button.

Also note that DRAFT and RETIRED kata cannot be submitted, but it's not the case here.
